esrally/driver/runner.py

```diff
@@ -2032,6 +2037,55 @@ def __repr__(self, *args, **kwargs):
         return "wait-for-snapshot-create"
 
 
+class WaitForCurrentSnapshotsCreate(Runner):
+    """
+    Waits until all currently running snapshots on a given repository have completed
+    """
+
+    async def __call__(self, es, params):
+        repository = mandatory(params, "repository", repr(self))
+        wait_period = params.get("completion-recheck-wait-period", 1)
+        es_info = await es.info()
+        es_version = tuple([int(part) for part in es_info["version"]["number"].split(".")[:2]])
+        api = es.snapshot.get
+        request_args = {"repository": repository, "snapshot": "_current", "verbose": False}
+
+        if es_version >= (8, 3):
+            request_params, headers = self._transport_request_params(params)
+            headers["Content-Type"] = "application/json"
+
+            # significantly reduce response size when lots of snapshots have been taken
+            # https://github.com/elastic/elasticsearch/pull/86269
```

Comment:
The reasoning behind putting the PR link here is to clarify that this feature (optionally excluding the large number of indices from the response) is only available from ES 8.3.0 onwards.
The TODO, later on, is about the support in elasticsearch-py since currently the `index_names` parameter in the version that we use is not supported.

I have improved and moved the comment to the beginning of the conditional in e482d72bacc74920de11b608bb2f70175299be3c to make this clearer. The TODO remained as is.